roles/logrotate/tasks/main.yml

```diff
@@ -29,8 +29,8 @@
     group: root
   when: item.1
   with_together:
-    - [ mesos, docker, zookeeper ]
-    - [ true, true, role == 'control' ]
```

Comment:
[These docs](https://docs.ansible.com/ansible/playbooks_loops.html#looping-over-parallel-sets-of-data) might help clarify what's going on here. Ansible loops over both sets of data at once. It might be easier to understand if I just use a 
```yaml
with_items:
  - condition: true
    component: docker
```
kind of thing, I didn't think of that when I simplified this role.